content/applications/studio/models_modules_apps.rst

```diff
@@ -212,16 +212,84 @@ Export and import customizations
 ================================
 
 When you do any customization with Studio, a new module named :guilabel:`Studio customizations` is
-added to your database.
+added to your database. You can export this module as a ZIP file using the :guilabel:`Studio Export`
+function. The module can then be imported into another Odoo database. This may be useful, for
+example, when setting up a new module or for training purposes.
 
-To export these customizations, go to :menuselection:`Main dashboard --> Studio --> Customizations
---> Export` to download a ZIP file containing all customizations.
+.. note::
+   Exporting and importing customizations in this way, rather than using the :doc:`standard Odoo
+   export and import<../essentials/export_import_data>` functions, means data is imported in a
```

Comment:
```suggestion
   export and import <../essentials/export_import_data>` functions, means data is imported in a
```
A space is required; otherwise, the :doc: hyperlink won't work.